pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: dartdoc
 # Run `grind build` after updating.
-version: 0.31.0
+version: 0.32.0
 description: A non-interactive HTML documentation generator for Dart source code.
 homepage: https://github.com/dart-lang/dartdoc
 environment:
-  sdk: '>=2.7.0 <3.0.0'
+  sdk: '>=2.8.0 <3.0.0'
```

Comment:
Confirming that we do only what this new version of dartdoc available to people on 2.8.0+.

Reply:
I didn't test it on anything less than 2.8.x which is why I updated the constraint.  The tests pass on 2.7 though so, reverted this change.